src/index.js

```diff
@@ -404,6 +430,7 @@ class CgSlider extends EventEmitter {
       case 'min':
       case 'max':
       case 'step':
+      case 'ariaValuetext':
```

Comment:
`aria-valuetext` attribute should be updated when new formatter function is set.